### PR TITLE
Add network bindings tests: remove and list

### PR DESCRIPTION
**Pull request by jmguzik**
Missing tests: network remove and list interactions.

### CODE REVIEW

**Comment by jmguzik** [comment]
CI flakes, please re-run.

File: pkg/bindings/test/networks_test.go [hunk 1]
```diff
@@ -116,4 +118,88 @@ var _ = Describe("Podman networks", func() {
 		Expect(err).To(BeNil())
 		Expect(data[0]["name"]).To(Equal(name))
 	})
+
+	It("list networks", func() {
+		// create a bunch of named networks and make verify with list
+		netNames := []string{"homer", "bart", "lisa", "maggie", "marge"}
```
Comment (TomSweeneyRedHat) [inline pkg/bindings/test/networks_test.go:124]: DOH!

**Comment by TomSweeneyRedHat** [comment]
LGTM
and happy green test buttons

File: pkg/bindings/test/networks_test.go (hunk 1 above)
Comment (jmguzik) [reply pkg/bindings/test/networks_test.go:124]: Inspired by volumes_test.go :)

**Comment by rhatdan** [comment]
Thanks @jmguzik it is always good to get more tests.

/lgtm

**Pull request merged**